pkg/crc/services/dns/dns_darwin.go

```diff
@@ -123,37 +125,37 @@ func waitForNetwork(hostResolv *network.ResolvFileValues) error {
 	return fmt.Errorf("Host is not connected to internet.")
 }
 
-// updateResolvConfFile updates the host's /etc/resolv.conf file with Instance IP.
-func updateResolvConfFile(instanceIP string, resolvConfFile string) (bool, error) {
-	// Get the current value of resolv.conf file
-	hostResolv, err := network.GetResolvValuesFromHost()
+// updateHostsConfFile updates the host's /etc/hosts file with Instance IP.
+func updateHostsConfFile(instanceIP string, clusterName string, baseDomain string, appsDomain string) (bool, error) {
```

Comment:
I think this function can be made both shorter and more generic, see https://github.com/cfergeau/crc/commit/173751dde0c550f5eb77974735bffbd9a5e84f79